Kickstarter-iOS/Views/Storyboards/SettingsPrivacy.storyboard

```diff
@@ -323,14 +322,14 @@
                                             <viewLayoutGuide key="safeArea" id="Ufg-5Z-5Nh"/>
                                         </view>
                                         <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="Uag-cG-Z0v">
-                                            <rect key="frame" x="0.0" y="1" width="375" height="50.5"/>
+                                            <rect key="frame" x="4" y="0.5" width="367" height="50.5"/>
                                             <constraints>
                                                 <constraint firstAttribute="height" constant="50" id="0wO-eK-0N1"/>
                                             </constraints>
                                         </button>
                                     </subviews>
                                     <constraints>
-                                        <constraint firstItem="Uag-cG-Z0v" firstAttribute="trailing" secondItem="ihf-gP-uZh" secondAttribute="trailingMargin" constant="16" id="7Ol-wf-rlh"/>
+                                        <constraint firstItem="Uag-cG-Z0v" firstAttribute="trailing" secondItem="ihf-gP-uZh" secondAttribute="trailingMargin" constant="12" id="7Ol-wf-rlh"/>
```

Comment:
Making the leading/trailing spaces smaller so that the button fits the screen. This should also be tested on iOS 10/11.